WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/SelectableShipmentItemRow.swift

```diff
@@ -22,40 +22,55 @@ struct SelectableShipmentItemRow: View {
                     }
             }
 
-            ProductImageThumbnail(productImageURL: viewModel.item.imageUrl,
-                                  productImageSize: Layout.imageSize,
-                                  scale: scale,
-                                  productImageCornerRadius: Layout.imageCornerRadius,
-                                  foregroundColor: Color(UIColor.listSmallIcon))
-            .overlay(alignment: .topTrailing) {
-                if viewModel.showQuantity {
-                    BadgeView(text: viewModel.item.quantityLabel,
-                              customizations: .init(textColor: .white, backgroundColor: .black),
-                              backgroundShape: badgeStyle)
-                    .offset(x: Layout.badgeOffset, y: -Layout.badgeOffset)
+            AdaptiveStack(spacing: Layout.horizontalSpacing) {
+                ProductImageThumbnail(productImageURL: viewModel.item.imageUrl,
+                                      productImageSize: Layout.imageSize,
+                                      scale: scale,
+                                      productImageCornerRadius: Layout.imageCornerRadius,
+                                      foregroundColor: Color(UIColor.listSmallIcon))
+                .overlay(alignment: .topTrailing) {
+                    if viewModel.showQuantity {
+                        BadgeView(text: viewModel.item.quantityLabel,
+                                  customizations: .init(textColor: .white, backgroundColor: .black),
+                                  backgroundShape: badgeStyle)
+                        .offset(x: Layout.badgeOffset, y: -Layout.badgeOffset)
+                    }
                 }
-            }
-            VStack(alignment: .leading) {
-                Text(viewModel.item.name)
-                    .bodyStyle()
-                Text(viewModel.item.detailsLabel)
-                    .subheadlineStyle()
-                AdaptiveStack(verticalAlignment: .lastTextBaseline) {
-                    Text(viewModel.item.weightLabel)
+
+                VStack(alignment: .leading) {
+                    Text(viewModel.item.name)
+                        .bodyStyle()
+                    Text(viewModel.item.detailsLabel)
```

Comment:
Should we check if the content is empty before rendering? That would avoid an empty line when there is no content:

<img width="354" height="147" alt="Screenshot 2025-07-16 at 10 03 48" src="https://github.com/user-attachments/assets/6e9f5c02-bd6e-48f9-a05c-67be2f42b930" />

Reply:
Good point - I'm good with keeping this as-is then.